### PR TITLE
[Bulk] Validation: Fix Alt Text issues (iis-docs)

### PR DESCRIPTION
**Pull request by reemsabaw1**
This PR fixes 76 missing alt text issues.

### CODE REVIEW

File: iis/configuration/system.applicationHost/applicationPools/add/recycling/periodicRestart/index.md [hunk 1]
```diff
@@ -55,11 +55,11 @@ The `<applicationPools>` collection is included in the default installation of I
 2. In the **Connections** pane, expand the server name, and then click **Application Pools**.
 3. In the **Application Pools** pane, select the application pool you want edit.
 4. In the **Actions** pane, click **Recycling...**   
-    [![](index/_static/image2.png)](index/_static/image1.png)
+    [![Screenshot of the Actions pane. Under- Edit Application Pool, Recycling is highlighted.](index/_static/image2.png)](index/_static/image1.png)
```
Comment (RedZephyr13) [inline iis/configuration/system.applicationHost/applicationPools/add/recycling/periodicRestart/index.md:58]: Let's take out the dash after "under."

File: iis/configuration/system.applicationHost/sites/site/ftpServer/directoryBrowse.md [hunk 2]
```diff
@@ -123,13 +123,13 @@ To support FTP publishing for your Web server, you must install the FTP service.
...
 3. In the **Home** pane, double-click **FTP Directory Browsing**.  
-    [![](directoryBrowse/_static/image10.png)](directoryBrowse/_static/image9.png)
+    [![Screenshot of the Home pane. F T P Directory Browsing is highlighted.](directoryBrowse/_static/image10.png)](directoryBrowse/_static/image9.png)
 4. Configure the following options: 
 
     - Click to select **UNIX** for the **Directory Listing Style**.
     - Click to select **Virtual directories**.
     - Click to select **Available bytes**.  
-        [![](directoryBrowse/_static/image12.png)](directoryBrowse/_static/image11.png)
+        [![Screenshot of the F T P Directory Browsing page. M S dash D O S is selected.](directoryBrowse/_static/image12.png)](directoryBrowse/_static/image11.png)
```
Comment (RedZephyr13) [inline iis/configuration/system.applicationHost/sites/site/ftpServer/directoryBrowse.md:132]: I think "D O S" should be "DOS."

File: iis/media/live-smooth-streaming-events/nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics.md [hunk 3]
```diff
@@ -152,15 +152,15 @@ A Content Manifest is essentially an XML-based playlist file that defines, and a
 
 As indicated in Figure 5, the fragments from the live stream are not copied and joined together; rather, the start and stop points in the original stream are referenced. When the player plays the clip, each stream is located and played in the order indicated in the Composite Manifest. Although Smooth Streaming files typically delivers fragments that are 2 seconds in duration, the Composite Manifest lets you start and end your clips within the boundaries of any given fragment. This means cuts are not constrained to be made at fixed two-second fragment intervals, but can be nearly frame-accurate.
 
-[![](nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics/_static/image20.jpg)](nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics/_static/image19.jpg)
+[![DIagram of the composite manifest for a highlights clip. It lets the user start and end clips within the boundaries of a given fragment.](nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics/_static/image20.jpg)](nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics/_static/image19.jpg)
```
Comment (RedZephyr13) [inline iis/media/live-smooth-streaming-events/nbc-online-video-monetization-and-highlights-creation-for-the-2010-winter-olympics.md:155]: Typo on "Diagram."

**Comment by reemsabaw1** [comment]
@Rick-Anderson @mcleblanc This PR fixes validation warnings/suggestions per a platform-wide push to increase content health scores. Please review at your convenience and sign off or merge if you approve, or let me know if I need to make any other changes. cc: @MonicaRush @snoviking

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@reemsabaw1  let's try to avoid merge conflicts. Make sure only one person is working on a file at a time. Not sure what caused this merge conflict.

**Pull request merged**